ebml/EbmlElement.h

```diff
@@ -328,7 +328,6 @@ class EBML_DLL_API EbmlCallbacks {
       ,DebugName(aDebugName)
       ,Context(aContext)
     {
-      assert(Creator!=nullptr);
```

Comment:
This change is unrelated to this MR.

Reply:
Quite related actually. GCC7 fails without this. Although it also fails for a different reason...

Reply:
That reason was discovered and placed in a different PR.